### ISSUE TITLE
Switch stipend proposals to git based and/or db storage.

### ISSUE THREAD

**Issue by starbelly** [erlef/infra-wg#51]
Currently stipend proposals go to email. This should be changed so that they go to a private repository or into the database. 

We had initially talked about it going to a private git repo, and I suppose it still can, but the data also has to go into the database. We are not able to create reports, etc right now and if we simply putting them in a git repo, we will still have that problem.

**Comment by benoitc** [comment]
How having the data in the database helps to create report? One point of having content is github is that we can easily share it with others.

**Comment by starbelly** [comment]
How do we sanely extract the numbers as an example?

**Comment by benoitc** [comment]
do we need to extract the number programatically? For which usage?

Right now stipends are handled by people that actually forward them by mail and discuss over mail about it. I am not sure we need more at the moment.  Have there been any request about it yet?

**Comment by starbelly** [comment]
Yes, this came up via sponsorship wg, perhaps @peerst can comment on this.

**Comment by starbelly** [comment]
@benoitc How about we have the best of both worlds like we do with board reports? That is, structured form of proposal goes into the database, but also a markdown version of the proposal gets opened as a PR on a private repo?

**Comment by peerst** [comment]
Not really fond of the everything is done via Github if it works or not.

**Comment by peerst** [comment]
Why e.g. not handle stipend emails with e.g.Trello?  In embedded WG we use Trello to develop our stipend proposals.  Look which WG is actually doing something in development proposal...

Development cost = 0

Trello boards can be emailed forwarded to (built in) and then proposals can be brought through a flexibly definable process.

**Comment by starbelly** [comment]
I am open to trello myself. I know we discussed this option before and I think the concern was more tooling, but for internal purposes and really at the board level, I think it could be fine. 

 I do think regardless of whether it is trello or github, the data should still go in the database. Can we compromise one that setup before discussing trello vs github? 
 
 In other words, can we agree that the data should go into the db AND also make use of trello or github for conversational purposes?
